bibliopixel/animation/circle.py

```diff
@@ -1,16 +1,16 @@
 from . animation import BaseAnimation
-from .. led import Circle
+from .. layout import Circle
 
 
 class BaseCircleAnim(BaseAnimation):
 
-    def __init__(self, led):
-        super().__init__(led)
+    def __init__(self, layout):
+        super().__init__(layout)
 
-        if not isinstance(led, Circle):
+        if not isinstance(layout, Circle):
             raise RuntimeError("Must use Circle with Circle Animations!")
```

Comment:
No need to change now, note to self that this would be clearer if "Must use bibliopixel.layout.Circle"